ui/hooks/gasFeeInput/useGasFeeInputs.js

```diff
@@ -133,6 +133,14 @@ export function useGasFeeInputs(
     Number(hexToDecimal(transaction?.txParams?.gas ?? '0x0')),
   );
 
+  const properGasLimit = Number(
```

Comment:
Why was the `?? '0x0'` needed? Is it possible to have a transaction without an `originalGasEstimate`?

Reply:
Yes @danjm, you're right, `?? '0x0` is not needed, it is possible to have a transaction without an `originalGasEstimate`. 
I will changed it immediately.
Thanks

Reply:
@danjm I resolved this issue.
You can see changes here: 365a71e
Is it okay now?